src/output.cpp

```diff
@@ -1796,7 +1797,8 @@ static void output_comment_multi(chunk_t *pc)
                && line[nwidx] != '*'    // block comment: skip '*' at end of line
                && (pc->flags.test(PCF_IN_PREPROC)
                    ? (  line[nwidx] != '\\'
-                     || (line[nwidx + 1] != 'r' && line[nwidx + 1] != '\n'))
+                     || (  line[nwidx + 1] != 'r'
```

Comment:
I think, 'r' should be replaced with '\r'

Reply:
the \ is one line above